backend/src/main/java/com/htecgroup/skynest/model/response/StorageContentResponse.java

```diff
@@ -0,0 +1,14 @@
+package com.htecgroup.skynest.model.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+public class StorageContentResponse {
+
+  private List<FolderResponse> folderResponseList;
```

Comment:
if it is list you can rename the variable with plural without the List suffix for example `folders` and `files`. It is more intuitive that way